Ghost filter/README.md

```diff
@@ -0,0 +1,22 @@
+# Applying Ghost filter on an image.
+
+Converting an image into ghost/negative image using OpenCv, Matplotlib and Numpy.
+
+## Steps:
+* Firstly imported the required libraries which are Numpy, Matplotlib and Cv2.
+* Read the input image/Real image using cv2
+
+### Methods Used
+* Used Bilateral Filter
+* Followed by Median Blur
+* Followed by Adaptive Threshold
+* And at last used Bitwise Xor
+* Finally converted the image into ghost filter image
+
+
+## Original Image
+<img src="Ghost filter/Images/Real Photo.jpg" height="300px">
```

Comment:
images still not rendered in the preview:

![image](https://user-images.githubusercontent.com/43691873/112718236-b2b29580-8f17-11eb-85d8-0b83048d48ea.png)

Reply:
It'll show once you merge it as it is working perfectly fine in my repo...
![pr](https://user-images.githubusercontent.com/73133055/112726050-5e241000-8f41-11eb-8081-0bc114238b0b.png)